docs/product-assembly_pam-on-openshift/src/main/asciidoc/ba-openshift-overview-con.adoc

```diff
@@ -0,0 +1,35 @@
+[id='ba-openshift-overview-con']
+= Overview
+If you have an OpenShift environment, you can deploy {PRODUCT} into this environment.
+
+In this solution, components of {PRODUCT} are deployed as separate OpenShift pods. You can scale each of the pods up and down individually, providing as few or as many containers as necessary for a particular component. You can use standard OpenShift methods to manage the pods and balance the load.
+
+The following key components are available as pods running in OpenShift:
+
+* {KIE_SERVER}, also known as _KIE Server_, is the infrastructure element that runs a process or a group of processes. All process logic runs on execution servers.
++
+A database backend is required for an execution server. You can provide a database server in another OpenShift pod or configure an execution server on OpenShift to use any other database server.
++
+You can freely scale up an {KIE_SERVER} pod, providing as many copies, running on the same host or different hosts, as necessary. As you scale a pod up or down, all its copies run the same processes and use the same database backend. OpenShift provides load balancing and a request can be handled by any of the pods.
++
+To run a different set of processes, deploy a separate {KIE_SERVER} pod, which can also be scaled up or down. You can have as many separate replicated {KIE_SERVER} pods as necessary.
++
+* {CENTRAL} is a web-based interactive environment for authoring processes. It also provides a management and monitoring console. You can use {CENTRAL} to develop processes, deploy processes to {KIE_SERVERS}, and monitor the execution.
++
+{CENTRAL} is a centralized application. However, you can configure it for high availability, where multiple pods run and share the same data.
++
+{CENTRAL} includes a Git repository that holds the source for the processes that you develop on it. It also includes a built-in Maven repository. Depending on configuration, {CENTRAL} can place the compiled processes (KJAR files) into the built-in Maven repository or into an external Maven repository.
++
+* {CENTRAL} Monitoring is a web-based management and monitoring console. It can manage deployment of processes to {KIE_SERVERS} and provide performance monitoring, but does not include authoring capabilities. Yiu can use this component to manage staging and production environments.
++
+* Smart Router is an optional layer between {KIE_SERVERS} and other components that interact with them. it is required if you want {CENTRAL} or {CENTRAL} Monitoring to interact with several different {KIE_SERVERS}. Also, when your environment includes many processes running on different {KIE_SERVERS}, Smart Router provides a single endpoint for all client applications. A client application can make a REST API call requiring any processes; Smart Router automatically determines which {KIE_SERVER} must be called for any particular request.
```

Comment:
it is required -> It is required (capitalization)
single endpoint for all client applications -> single endpoint to all client applications
requiring any processes -> requiring any process